spacebrew.js

```diff
@@ -20,7 +20,13 @@ var path = require('path')
     , serveStatic = require('serve-static')
     , http = require('http')
     , finalhandler = require('finalhandler')
+    , AJV = require('ajv')
+    , fs = require('fs')
     ;
+
+//setup validator
+var ajv = AJV();
+var validate = ajv.compile(JSON.parse(fs.readFileSync('schema.json')));
```

Comment:
Maybe. In working on the electron version, I'm realizing we need to have some better mgmt of things outside scripts might want to manage, e.g. having some way of closing/editing server after open. With this, it's only an issue if there's something weird about a) when the file is loaded and b) where it's loaded from. Should it be __dirname + 'shema.json'??